Compilation fails with latest Tribblix patches due to undefined 'thread_t'. Including "thread.h" here fixes that for now.

diff -Nru jdk25u-jdk-25-36.orig/src/hotspot/os/solaris/osThread_solaris.hpp jdk25u-jdk-25-36/src/hotspot/os/solaris/osThread_solaris.hpp
--- jdk25u-jdk-25-36.orig/src/hotspot/os/solaris/osThread_solaris.hpp	2025-08-17 11:03:54.987465970 +0200
+++ jdk25u-jdk-25-36/src/hotspot/os/solaris/osThread_solaris.hpp	2025-08-17 11:05:02.227576984 +0200
@@ -28,6 +28,7 @@
 #include "runtime/osThreadBase.hpp"
 #include "suspendResume_posix.hpp"
 #include "utilities/globalDefinitions.hpp"
+#include "thread.h"
 
 class OSThread : public OSThreadBase {
   friend class VMStructs;
